roles/ipasmartcard_client/README.md

```diff
@@ -0,0 +1,112 @@
+ipasmartcard_client role
+========================
+
+Description
+-----------
+
+This role allows to configure IPA clients for Smart Card authentication.
+
+**Note**: The ansible playbook and role requires deployed IPA clients.
```

Comment:
Do the host have to be enrolled for this to work?

Reply:
Yes, the smartcard setup of the client requires a enrolled client as it is using kerberos and configured services on the client.